pkg/kubefed2/join.go

```diff
@@ -440,6 +447,19 @@ func createRBACSecret(hostClusterClientset, joiningClusterClientset client.Inter
 		glog.V(2).Infof("Created role and binding for service account: %s in joining cluster: %s",
 			saName, joiningClusterName)
 
+		glog.V(2).Infof("Creating health check cluster role and binding for service account: %s in joining cluster: %s", saName, joiningClusterName)
+
+		err = createHealthCheckClusterRoleAndBinding(joiningClusterClientset, saName, namespace, joiningClusterName,
+			dryRun)
```

Comment:
(No action required) Seems strange to break here given that the line above would be longer.